changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+ <changeSet author="Marc Templeton" id="2">
+    <tagDatabase tag="v1.0.0" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="3">
+    <createTable tableName="paymentRequests">
+      <column name="paymentRequestId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="paymentRequestsId_pkey" />
+      </column>
+      <column name="agreementNumber" type="VARCHAR(20)" />
+      <column name="contractNumber" type="VARCHAR(10)" />
+      <column name="correlationId" type="VARCHAR(36)" /> <!-- interesting 1.23 rather tnan uuid -->
+      <column name="currency" type="VARCHAR(3)" />
+      <column name="deliveryBody" type="VARCHAR(4)" />
+      <column name="dueDate" type="VARCHAR(10)" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="marketingYear" type="INTEGER" />
+      <column name="referenceId" type="UUID" /> <!-- interesting 1.23 -->
+      <column name="schedule" type="VARCHAR(3)" />
+      <column name="sitiAgriInvoiceNumber" type="VARCHAR(20)" />
```

Comment:
I have decided to create a separate field to store the original (reverse engineered) invoice number that this payment request would have originally had before we enrich and split it. Is this sensible firstly?

Does this make this dependent on SitiAgri, what if a new data source was used?

Reply:
Commented on this above as missed this comment.  Oops.

See what I've put above as there a few options for this.  We will have many statements that are independent of Siti Agri so I don't think we should tightly couple the data together in this table.  The question is should we save it at all or should we calculate it when we need it.  Is there a scenario the original invoice could change that would mean saving it is a risk?  I don't think so..

Reply:
Have fleshed this out into its own mapping table